docs/release-notes/2.39.0.md

```diff
@@ -0,0 +1,74 @@
+
+### Query Analytics - Meta information
+
+- Why: QAN is a useful tool for filtering data and identifying problematic queries. However, it can be difficult to determine which services, nodes, and other labels are associated with the identified queries.
+
+- What: This new feature assists the user in viewing all labels associated with the chosen query (metadata) to streamline the process of identifying and resolving issues.
+
+
+
+## Improvements
+
+- [PMM-6279](https://jira.percona.com/browse/PMM-6279) - [QAN]: Query metadata not visible in new UI
+- [PMM-10270](https://jira.percona.com/browse/PMM-10270) -[Backup Management]: Restore on a new replica set cluster
+- [PMM-11999](https://jira.percona.com/browse/PMM-11999) - [Advisors]: Expose check family field in advisors API
+- [PMM-12257](https://jira.percona.com/browse/PMM-12257) - [Advisors]: The **Advisors** page now includes a new **Technology** column, showing the database type for each advisor in the table. This helps you easily differentiate between the MongoDB, MySQL and PostgreSQL	advisors.
+- [PMM-11637](https://jira.percona.com/browse/PMM-11637) - In PMM 2.39.0, we updated the interface to reflect the new PMM logo as per the new branding.
+- [PMM-11206](https://jira.percona.com/browse/PMM-11206) - [Alerting]: Enhanced the **Add rule template** dialog box to support YAML content for multiple alert templates. Alert templates added in bulk are displayed individually on the **Alert rule templates** page.
+
+ 
+## Bugs Fixed
+
+- [PMM-12151](https://jira.percona.com/browse/PMM-12151) - [QAN]: Explain plan not displayed if database name is missing in the query
+- [PMM-10391](https://jira.percona.com/browse/PMM-10391) - PMM did not work with external clickhouse versions greater than 22.5.2. The issue has been resolved now.
+- [PMM-11658](https://jira.percona.com/browse/PMM-11658) - pmm-agent crashes simultaneously on all nodes
+- [PMM-11992](https://jira.percona.com/browse/PMM-11992) - Database name is ignored while adding any Postgres
+- [PMM-12161](https://jira.percona.com/browse/PMM-12161) - "MongoDB Cluster Summary" - too many "Config Services"
+- [PMM-12231](https://jira.percona.com/browse/PMM-12231) - PMM disallows to install, update and remove plugins
+- [PMM-12263](https://jira.percona.com/browse/PMM-12263) - Wrong Replication Lag by Set values if RS is PSA - MongoDB Cluster Summary
+- [PMM-12280](https://jira.percona.com/browse/PMM-12280) - Service Summary Panel on Home Dashboard Seems Broken
+
+
+
+## Known issues
+
+**PMM does not allow to install, update or remove plugins**
+
+[PMM-12231](https://jira.percona.com/browse/PMM-12231) - In the recent versions of PMM, users have encountered issues with installing, updating and removing plugins from PMM. The cause of this issue is the incorrect permissions assigned to the `/srv/grafana/plugins` directory. These permissions are preventing the grafana component from accessing the directory.
```

Comment:
```suggestion
[PMM-12231](https://jira.percona.com/browse/PMM-12231) - In the recent versions of PMM, users have encountered issues with installing, updating and removing plugins from PMM. The cause of this issue is the incorrect permissions assigned to the `/srv/grafana/plugins` directory. These permissions are preventing the grafana component from writing to the directory.
```

Reply:
Why this is Known issue - if we fix this in the current release??

Reply:
@atymchuk If this is fixed. Why do we have a workaround?